browser/src/control/Control.Notebookbar.js

```diff
@@ -332,12 +332,14 @@ L.Control.Notebookbar = L.Control.extend({
 		if (button.unoCommand && !isUnoCommand)
 			button.unoCommand = '.uno:' + button.unoCommand;
 
+		var imgUrl = button.imgurl.split('/')[1];
```

Comment:
this will break custom icons from integrator

Reply:
You are right this is wrong. Most probably we send wrong image url via postMessage frame here:
https://github.com/CollaboraOnline/online/blob/500df64292e53fe07c92d492511ed82bfcb8b546/browser/html/framed.doc.html#L253

We just send lc_save.svg  instead of images/lc_save.svg or leave empty. If it is empty it automatically uses the uno command's default icon.

Reply:
Great, could you fix it then?

Reply:
and check if SDK tell something about it? do we have any wrong information there?

Reply:
I'm testting Szymon. Let me write down my detections.

We don't let the handle post message without imgurl here https://github.com/CollaboraOnline/online/blob/3d6065175382afd3b83c4f5e9221677b225942e4/browser/src/map/handler/Map.WOPI.js#L445 
so it never be undefined in this case.

If I change the images/lc_save.svg  to lc_save.svg  tabbed view is ok but compact view is nok ok. 
In the compact view we should see img style="background: url(&quot;images/lc_save.svg&quot;) center center no-repeat;" 

So I'll check SDK then change frame.html, compact or tabbed view url handling...

Reply:
@eszkadev I solved it. I didn't touch the compact view and frame.doc.html 

I just forced the tabbed view to use default button icon, if command is an uno command and url is an internal path. So It won't break if icon url is absolute path "http...."